src/components/image/Image.tsx

```diff
@@ -6,24 +6,11 @@ type Image = {
   width: number;
   height: number;
   altText: string;
-  caption: string;
-  attribution: string;
   alignment: "left" | "center" | "right";
 };
 
-// const [isHovering, setIsHovering] = useState(false)
-
-const Images: FC<Image> = ({
-  image,
-  width,
-  height,
-  altText,
-  caption,
-  attribution,
-  alignment,
-}) => {
+const Images: FC<Image> = ({ image, width, height, altText, alignment }) => {
```

Comment:
Include sizes and style props (see https://github.com/distributeaid/next-website-v2/pull/211/files#diff-da94d5a2e703325b54c2f3822b9a488da97997c511acb3d5e55a73f9703ac731)

Reply:
Heard, those changes should be reflected in [58f993a](https://github.com/distributeaid/next-website-v2/pull/243/commits/58f993ab2236dd5aec09f382bc071afe1ff76901)